tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/connect/AMQPBridgeTest.java

```diff
@@ -54,17 +54,6 @@ protected ActiveMQServer createServer() throws Exception {
       return createServer(AMQP_PORT, false);
    }
 
-   @Test
-   public void testsSimpleConnect() throws Exception {
```

Comment:
This test was not doing anything at this point..

I had use for it during dev, but as things progressed it is not needed any longer.